app/views/content/what-is-a-pgce.md

```diff
@@ -70,6 +70,8 @@ You can do this through a school-led, university-led, or an apprenticeship teach
 
 You can do a full or part-time PGCE course.
 
+As part of selecting your teacher training course, you will need to decide if you want to train to teach at a primary or secondary level. [Learn about deciding who to teach](/train-to-be-a-teacher/who-do-you-want-to-teach).
```

Comment:
```suggestion
As part of selecting your course, you will need to decide if you want to train to teach at a primary or secondary level. [Learn about deciding who to teach](/train-to-be-a-teacher/who-do-you-want-to-teach).
```
just cos this section is mainly talking about 'pgce courses' rather than 'teacher training courses'? makes it a bit more consistent, and shorter